docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md

```diff
@@ -138,6 +138,23 @@ Finally, change the _loris root_ directory owner to the _lorisadmin_ account
 sudo chown lorisadmin.lorisadmin /var/www/loris
 ```
 
+## Creating the lorisadmin user
+
+```bash
+# Create lorisadmin user and group
+# Give lorisadmin `sudo` permission. This is required for the install process
+# in order to automatically generate Apache configuration files.
+# Sudo privileges can be revoked once the install is completed.
+sudo useradd -U -m -G sudo -s /bin/bash lorisadmin
```

Comment:
@johnsaigle I know this subject has come up again and again... but im really not a fan of having lorisadmin be a sudoer !!

I understand we need sudo to do the initial install and configuration but cant we tell user to either : 
- run the following command with sudo credentials
or
- add lorisadmin to sudoers but then remove it later on

Reply:
ALSO this PR conflicts with lines 108 to 140... can you check if the content is the same or different ?

Reply:
I'll delete what's above

Line 111-112 mentions that you can remove sudo after the install if that's what you mean

Reply:
yeah... but I'd want it to be much more clear wording like SHOULD be revoked...